src/smk/component/alert/component-alert.html

```diff
@@ -0,0 +1,15 @@
+<div class="smk-modal-mask">
+  <div class="smk-modal-wrapper">
+    <div class="smk-modal-container">
+      <div class="smk-modal-body">
+        <slot name="alertBody"></slot>
+      </div>
+      <div class="smk-modal-footer">
+        &nbsp;
```

Comment:
This non-breaking space should be removed as well.

Reply:
The remaining non-breaking space is deliberate, to give us desired vertical spacing. If it's removed, the alert looks like:

<img width="466" alt="Screen Shot 2021-11-02 at 8 28 44 AM" src="https://user-images.githubusercontent.com/1911540/139878016-79c1480d-413b-4654-941c-c7bdbaaf7df3.png">

I wasn't sure how to address it in CSS, but on further thought, setting a fixed height on the "smk-modal-footer" div seems like a reasonable fix as we probably wouldn't expect variable text (or any text) in that area. Actually, on even further thought, min-height seems safer, in case we do get text added to the footer.